flashcard.js

```diff
@@ -18,15 +23,30 @@ export default class Flashcard extends React.Component {
     for (var pair of formData.entries()) {
       cardObj[pair[0]] = pair[1]
     }
-    const flashCardStateCopy = this.state.flashcard.slice(0)
+    const flashCardStateCopy = this.state.flashcards.slice(0)
     flashCardStateCopy.push(cardObj)
-    this.setState({flashcard: flashCardStateCopy})
+    this.setState({flashcards: flashCardStateCopy})
     cardForm.reset()
   }
 
+  cardsOnClick (event) {
+    this.setState({view: 'Cards'})
+  }
+
+  createOnClick (event) {
+    this.setState({view: 'New'})
+    console.log.bind(this.state)
+  }
+
   render() {
+    const {view} = this.state
+    const {flashcards} = this.state
     return (
-      <FlashcardBuild saveOnClick={this.saveOnClick}/>
+      <div>
+        <Navigation cardsOnClick={this.cardsOnClick} createOnClick={this.createOnClick}/>
+        <CreateCard saveOnClick={this.saveOnClick} view={view}/>
```

Comment:
Instead of rendering both the `CreateCard` and the `ViewCards`, do your conditional rendering based on `this.state.view` here. That will simplify your `CreateCard` and `ViewCards` components quite a bit.